MkDocs/apm_8.8/docs/matrix.md

```diff
@@ -0,0 +1,100 @@
+- Make sure the exercise [Understand Health and Predict - Utilities Models](hpu_models.md) is done, so that Sample ST(Substation Transformer) assets have the EOL/Risk/Health scores
+- Members of the EUSCORING security group can configure settings for the matrix(exercise 3), such as which version of the matrix is the default version and the colors and placement of cells.
+
+Please note that the MAS Worldwide (WW) demo environment is NOT a suitable environment for this lab.  The WW demo environment is shared, and making ANY changes to that environment will impact other users’ ability to demonstrate MAS.
+
+## Introduction
+
+This lab is intended to demonstrate the Matrix view capability in HPU which is another view besides the existing Grid view, Map view, Charts View. It allows Reliability engineer to esaily idenfity the high risk assets from a matrix distribution view and take actions based on the distributions.
+Reliability engineer can also easily config the different matrix versions, like to change the default version, disable version, change the matrix cells' color(category) etc.
+
+
+
+##  Navigate to Health and Predict - Utilities
+
+`Action:`
+
+1. Navigate to Health and Predict - Utilities (HPU). From the main MAS page, select the `Industry solutions` tab.
+![setup assets](/img/apm_8.7/AIO 1.png)
+
+2. Then click `Health and Predict – Utilities` menu.
+![setup assets](/img/apm_8.7/AIO 2.png)	 
+
+
+
+## View Matrix for Criticality and End of life, Criticality and Risk, Criticality and Health
+
+`Action:`
+1. From the main grid view, click the Matrix tab, you will see the matrix, each cell contains the assets count which fall in the x-axis and y-axis range.
+   For example, the red cell in the top right corner shows the assets that End of Life is high and Criticality is also High which means high risks.
+   Besides cells, there are also several cards which is summarized count for the different categories of assets, normally High is for assets that need to take action in high priority.
+
+    ![matrix view](./img/matrix_home.png)
+
+2. If there are too many assets in one cells or category, you can also use filter to reduce the counts. The followings are the detailed steps:
+   - click the filter icon, then select `Type`
+   - From the pop-up, find `Substation Transformers`, select it, and press `OK`.  It may be necessary to use the search capability or scroll.
+   - Finally, press `Apply`, the Matrix count 
+
+3. The default matrix version is Criticality and End of life, you can also click the dropdown on the top right corner to change to other matrix versions. 
+
+    ![matrix view](./img/matrix_versions.png)
+
+
+## View the drilled in assets list for one distribution
+`Action:`
+1. Click any cell of Matrix or any card with the blue text of number, you can see the detailed assets list.
+  ![matrix view](./img/matrix_home_click.png)
```

Comment:
Wont the more likely path be for them to click on the high risk value card
<img width="1670" alt="matrix_home_new HighRiskValue Card" src="https://user-images.githubusercontent.com/60844296/184695116-91c81c03-2189-40b5-8622-c495397d751f.png">
.

Reply:
fixed.